### PR TITLE
Remove unneeded "io" folder in parselib.

### PR DESCRIPTION
**Pull request by android-usb**
Pull "stream" and "wav" folders up a level.

### CODE REVIEW

File: samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp [hunk 1]
```diff
@@ -23,8 +23,8 @@
 #include <android/log.h>
 
 // parselib includes
-#include <io/stream/MemInputStream.h>
-#include <io/wav/WavStreamReader.h>
+#include <stream/MemInputStream.h>
```
Comment (philburk) [inline samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp:26]: or change "io" to "parselib"

File: samples/parselib/src/main/cpp/CMakeLists.txt [hunk 2]
```diff
@@ -42,15 +42,15 @@ add_library( # Sets the name of the library.
 
         # Provides a relative path to your source file(s).
         # io/stream
```
Comment (philburk) [inline samples/parselib/src/main/cpp/CMakeLists.txt:44]: **io**/stream?

File: samples/parselib/src/main/cpp/CMakeLists.txt [hunk 3]
```diff
@@ -42,15 +42,15 @@ add_library( # Sets the name of the library.
 
         # Provides a relative path to your source file(s).
         # io/stream
-        ${CMAKE_CURRENT_LIST_DIR}/io/stream/FileInputStream.cpp
-        ${CMAKE_CURRENT_LIST_DIR}/io/stream/InputStream.cpp
-        ${CMAKE_CURRENT_LIST_DIR}/io/stream/MemInputStream.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/stream/FileInputStream.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/stream/InputStream.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/stream/MemInputStream.cpp
         # io/wav
```
Comment (philburk) [inline samples/parselib/src/main/cpp/CMakeLists.txt:48]: io/wav

File: samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp (hunk 1 above)
Comment (android-usb) [reply samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp:26]: I don't see what you are referring to.

File: samples/parselib/src/main/cpp/CMakeLists.txt (hunk 2 above)
Comment (philburk) [reply samples/parselib/src/main/cpp/CMakeLists.txt:44]: This change remove "io/" from the path but it is still in the comment on line 44.
Should that be "parselib/stream".

File: samples/parselib/src/main/cpp/CMakeLists.txt (hunk 3 above)
Comment (philburk) [reply samples/parselib/src/main/cpp/CMakeLists.txt:48]: Same as above about "io/".

File: samples/parselib/src/main/cpp/CMakeLists.txt (hunk 2 above)
Comment (android-usb) [reply samples/parselib/src/main/cpp/CMakeLists.txt]: done.

**Pull request merged**